Userland/Utilities/pkg/AvailablePort.cpp

```diff
@@ -105,24 +115,31 @@ ErrorOr<int> AvailablePort::update_available_ports_list_file()
     return loop.exec();
 }
 
-static ErrorOr<String> extract_port_name_from_column(Markdown::Table::Column const& column, size_t row_index)
+struct NameAndPath {
+    String name;
+    String path;
+};
+
+static ErrorOr<NameAndPath> extract_port_name_and_path_from_column(Markdown::Table::Column const& column, size_t row_index)
```

Comment:
"pkg: Parse and store Port paths separately" looks like it should be its own commit if done like this.

Note that `name` and `path` should **always** be the same anyways. If not, that's an issue either with `AvailablePorts.md` or the `package.sh` script (and its location) itself.

TL;DR: Just use `name` for building the path.

Reply:
I can't spot any as of now, but I only gave it a cursory look. In any case, that's an issue with that particular ports script, and nothing that `pkg` needs to care about.

Reply:
Alright then.